app/connectors/ThirdPartyApplicationConnector.scala

```diff
@@ -65,6 +66,26 @@ abstract class ThirdPartyApplicationConnector(config: ApplicationConfig, metrics
   def update(applicationId: ApplicationId, request: UpdateApplicationRequest)(implicit hc: HeaderCarrier): Future[ApplicationUpdateSuccessful] = metrics.record(api) {
     http.POST[UpdateApplicationRequest,ErrorOrUnit](s"$serviceBaseUrl/application/${applicationId.value}", request).map(throwOr(ApplicationUpdateSuccessful))
   }
+  
```

Comment:
Need to remove logs in mirrored "email" function as we could still log emails if called.

Reply:
👍